src/assets/custom/css/_sass/_training-hero.scss

```diff
@@ -0,0 +1,149 @@
+.training-hero__background-image {
+  background-size: cover;
+  opacity: 0.9;
+  position: relative;
+  width: 100%;
+
+  @include small {
+    height: 216.27px;
+  }
+
+  @include medium-and-large {
+    height: 433px;
+  }
+
+  @include extra-large {
+    height: 519.5px;
+  }
+}
+
+$background-block-height: ($vertical-overlap *  4);
+.training-hero__lower-background-block {
+  background-color: $slate;
+  position: relative;
+
+  @include small {
+    height: 2px + $background-block-height;
+  }
+
+  @include medium {
+    height: 6px + $background-block-height;
+  }
+
+  @include large {
+    height: 16px + $background-block-height;
+  }
+
+  @include extra-large {
+    height: 28px + $background-block-height;
+  }
+}
+
+.training-hero__content-container {
+  max-width: 50%;
+  position: relative;
+  color: white;
+  transform: translateY(-650px);
+
+  @include small {
+    padding-top: 85px;
+  }
+  
+  @include small-and-medium {
+    @include max-width--out-there;
```

Comment:
This would set a max width larger than our medium breakpoint. We can take a look at this.